### DISCUSSION TITLE
Find elements in an old Java application

### DISCUSSION THREAD

**Discussion by Abun1407** [gaozhao1989/pyjab#69] [Q&A]
Hi,

I am trying to find this button. It does not have a name.

![image](https://github.com/gaozhao1989/pyjab/assets/124797995/73fb6107-0624-41d6-aeff-8764f0c3d1f4)

Is it possible to find it using the combination of "Object Depth" and "Index in parent" properties? Or XPATH?

I tried this, but no luck

`planButton=jabdriver.find_element_by_object_depth(13).find_element_by_index_in_parent(0)
print(planButton.get_element_information())`

Its not printing the information of that button.

Kindly assist here.

**Comment by gaozhao1989** [comment]
Use xpath 
Sample:
//scroll pane[@indexinparent=0]/panel[@indexinparent=1]/text [@indexinparent=5]
